var/spack/repos/builtin/packages/plumed/package.py

```diff
@@ -21,9 +21,16 @@ class Plumed(AutotoolsPackage):
     and C/C++ codes.
     """
     homepage = 'http://www.plumed.org/'
-    url = 'https://github.com/plumed/plumed2/archive/v2.5.0.tar.gz'
-
+    url = 'https://github.com/plumed/plumed2/archive/v2.5.3.tar.gz'
+    git = 'https://github.com/plumed/plumed2.git'
+
+    version('develop', branch='master')
```

Comment:
```suggestion
    version('master', branch='master')
```
See #1983

Reply:
The change seems reasonable.

Reply:
The change you suggest is reasonable. Do you need any action from my side to finalize the commit?